lib/membrane_http_adaptive_stream/sink_bin.ex

```diff
@@ -75,6 +75,14 @@ defmodule Membrane.HTTPAdaptiveStream.SinkBin do
         description: """
         Encoding type determining which payloader will be used for the given stream.
         """
+      ],
+      track_name: [
+        spec: String.t() | nil,
+        default: nil,
+        description: """
+        Name of multimedia track that will be used to name media playlist and related header and segments files.
+        Therefore it is expected to be a vaild filename (eg. cannot contain slashes and other forbidden characters).
```

Comment:
If you updated the other description, don't forget about this one

Reply:
Updated with the same content.